### PR TITLE
Remove decompress dependency

### PR DESCRIPTION
**Pull request by vvscode**
@ewnd9  sorry, I was late for a few minutes

This one remove redundant dependency and fix tests for node 0.10 / 0.12 ( but I'll leave matrix which allow them to be failed )

### CODE REVIEW

File: test/_helpers.js [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
+import fs from 'fs';
+import path from 'path';
+import Zip from 'jszip';
 
 export const addCards = (apkg, list) => list.forEach(({front, back}) => apkg.addCard(front, back));
 
-export const unzipDeckToDir = (pathToDeck, pathToUnzipTo) => decompress(pathToDeck, pathToUnzipTo);
\ No newline at end of file
+export  function unzipDeckToDir(pathToDeck, pathToUnzipTo) {
+    !fs.existsSync(pathToUnzipTo) && fs.mkdirSync(pathToUnzipTo);
```
Comment (ewnd9) [inline test/_helpers.js]: Please add [`mkdirp`](https://github.com/substack/node-mkdirp) and change to 

```js
mkdirp.sync(pathToUnzipTo)
```

File: test/_helpers.js [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
+
+    const zipContent = fs.readFileSync(pathToDeck);
+    const zip = new Zip();
+    return zip
+            .loadAsync(zipContent, { createFolders: true })
+            .then(() => Promise.all(Object.keys(zip.files).map(i => {
+                        const file = zip.files[i];
+                        let filePath = path.join(pathToUnzipTo, file.name);
```
Comment (ewnd9) [inline test/_helpers.js]: `let` => `const`

File: test/_helpers.js [hunk 3]
```diff
@@ -1,8 +1,23 @@
 import 'babel-register';
```
Comment (ewnd9) [inline test/_helpers.js]: I missed that in the previous :grin: `babel-register` and `babel-polyfill` should be imported only at entry points (it's not an error, just a redundancy)

Comment (vvscode) [inline test/_helpers.js]: Looks like I'll find many new and useful with your PR's )) thx

Comment (ewnd9) [inline test/_helpers.js]: Always happy to share knowledge :fist:

**Comment by vvscode** [comment]
@ewnd9  thx for your feedback. It's very useful

**Comment by ewnd9** [comment]
:tada: Merged

**Comment by ewnd9** [comment]
I've pushed https://github.com/ewnd9/anki-apkg-export/commit/1e828a7df7e08fb3647693f9e8ab022a26bf4afc with updated linting to prevent nitpicking

**Pull request merged**